api/_lib/script/build.ts

```diff
@@ -18,6 +18,12 @@ async function main() {
     { recursive: true, force: true }
   );
 
+  await fs.cp(
+    join(repoRoot, 'packages', 'fs-detectors', 'src', 'monorepos', 'logos'),
```

Comment:
Lets move this out of `src` since these images are not source files

Reply:
👍